docs/main.css

```diff
@@ -0,0 +1,249 @@
+* {
+    margin: 0px;
+    padding: 0px;
+}
+
+/* Header Content */
+header {
+    text-align: center;
+    padding-top: 95px;
+    border: 1px solid #979797;
+    background: radial-gradient(100% 50%, #E9A571 0%, #EE7C5A 82%);
+}
+
+#headline {
+    font-family: "Arial Black";
+    color: #FFFFFF;
+    font-size: 72px;
+    line-height: 101px;
+}
+
+#subheadline {
+    font-family: "Arial Black";
+    color: #916246;
+    font-size: 24px;
+    line-height: 33px;
+}
+
+.linkContainer {
+    min-width: 200px;
+    margin-top: 92px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    padding: 20px 0px 10px;
+    background-color: rgba(174,174,174,0.65);
+}
+
+#npmpackage, #npmversion {

```

Comment:
We will want to minimize the use of IDs and try to use classes as much as possible.

What other ways are there to target this selector?